File: app/src/main/java/com/otraempresa/android/example/RequestCashAsyncTokenAsyncTask.java
package com.otraempresa.android.example;

import android.content.Context;

import com.kushkipagos.android.KushkiException;
import com.kushkipagos.android.Transaction;

public class RequestCashAsyncTokenAsyncTask extends AbstractRequestCashTokenAsyncTask {

    RequestCashAsyncTokenAsyncTask(Context context) {
        super(context);
    }

    @Override
    protected Transaction requestCashAsyncToken(String name) throws KushkiException {
        return kushki.requestCashToken(name,
                "Su","1237254","CC","test@test.com",100,"USD");
    }

}

File: app/src/main/java/com/kushkipagos/kushkidemo/services/RequestCashAsyncTokenAsyncTask.kt
package com.kushkipagos.kushkidemo.services

import android.content.Context
import com.kushkipagos.exceptions.KushkiException
import com.kushkipagos.kushkidemo.contracts.AbstractRequestCashTokenAsyncTask
import com.kushkipagos.models.Transaction

class RequestCashAsyncTokenAsyncTask: AbstractRequestCashTokenAsyncTask {
    constructor(context: Context): super(context)

    @Throws(KushkiException::class)
    override fun requestCashAsyncToken(name: String): Transaction {
        return _kushki.requestCashToken(name,
            "Su","1237254","CC","test@test.com",100.0,"USD")
    }
}